### PR TITLE
Add Kaixiang MegaBric CSI Driver

### PR DESCRIPTION
**Pull request by wangl0320**

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [ionir ](https://github.com/ionir-cloud) | `ionir` | v1.2 | A Container Storage Interface (CSI) Driver for [ionir](https://www.ionir.com/) Kubernetes-Native Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Cloning
 [JuiceFS](https://github.com/juicedata/juicefs-csi-driver) | `csi.juicefs.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for JuiceFS File System | Persistent | Read/Write Multiple Pod | Yes | 
 [kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | 
+[KaiXiangTech MegaBric](https://github.com/kaixiangtech/csi-driver-flexblock) | `flexblock.csi.kaixiangtech.com` | 21.01.01.1, 21.03.04.1 | A Container Storage Interface (CSI) plugin for KaiXiangTech MegaBric Storage | Persistent | Read/Write MultiplePods | Yes | Raw Block, Expansion, Cloning
```
Comment (xing-yang) [inline book/src/drivers.md:69]: Please add a README to the github repo describing how to use the driver and what it supports.

Also column 3 is for CSI spec versions supported by your CSI driver.

Add a space betwee "Multiple" and "Pods".

**Comment by xing-yang** [comment]
Please sign Linux Foundation CLA.  Click "Details" next to the error for instructions.

File: book/src/drivers.md (hunk 1 above)
Comment (wangl0320) [reply book/src/drivers.md]: update Version v1.6.0 .

Comment (wangl0320) [reply book/src/drivers.md]: update README file into github repo and tag v1.6.0 version.

Comment (wangl0320) [reply book/src/drivers.md]: commit add  a space betwee "Multiple" and "Pods" and version v1..6.0

File: book/src/drivers.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [ionir ](https://github.com/ionir-cloud) | `ionir` | v1.2 | A Container Storage Interface (CSI) Driver for [ionir](https://www.ionir.com/) Kubernetes-Native Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Cloning
 [JuiceFS](https://github.com/juicedata/juicefs-csi-driver) | `csi.juicefs.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for JuiceFS File System | Persistent | Read/Write Multiple Pod | Yes | 
 [kaDalu](https://github.com/kadalu/kadalu) | `org.kadalu.gluster` | v0.3 | A CSI Driver (and operator) for GlusterFS | Persistent | Read/Write Multiple Pods | Yes | 
+[KaiXiangTech MegaBric](https://github.com/kaixiangtech/csi-driver-flexblock) | `flexblock.csi.kaixiangtech.com` | v1.6.0 | A Container Storage Interface (CSI) plugin for KaiXiangTech MegaBric Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion, Cloning
```
Comment (xing-yang) [inline book/src/drivers.md:69]: The version column refers to the CSI spec version, not the CSI driver version. The latest CSI spec version is 1.5: https://github.com/container-storage-interface/spec/releases/tag/v1.5.0

Comment (wangl0320) [reply book/src/drivers.md]: we test success in new version
change support version v1.5.0

**Comment by xing-yang** [comment]
Can you squash your commits into 1?

**Comment by wangl0320** [comment]
> Can you squash your commits into 1?

change to one commit

**Pull request merged**